src/test/java/seedu/storemando/testutil/TypicalItems.java

```diff
@@ -75,6 +75,6 @@ public static StoreMando getTypicalStoreMando() {
     }
 
     public static List<Item> getTypicalItems() {
-        return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE));
+        return new ArrayList<>(Arrays.asList(MILK, BREAD, OIL, CADBURY, NESCAFE, LIPTON, PRATA));
```

Comment:
What's the set of names we following from now on? For the original ab3 it was ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE. What's the pattern for ours?

Reply:
It will be MILK, BREAD, OIL, CADBURY, NESCAFE, LIPTON and PRATA respectively.